samples/LowAllocationWebServer/LowAllocationWebServerLibrary/TcpConnectionFormatter.cs

```diff
@@ -97,7 +97,7 @@ public void Dispose()
         }
         int WriteChunkPrefix(Span<byte> chunkPrefixBuffer, int chunkLength)
         {
-            if (!PrimitiveFormatter.TryFormat(chunkLength, chunkPrefixBuffer, out var written, 'X', EncodingData.InvariantUtf8))
+            if (!PrimitiveFormatter.TryFormat(chunkLength, chunkPrefixBuffer, out var written, 'X', TextEncoder.InvariantUtf8))
```

Comment:
Technically, it does not. This was just a blind patching up of the code after my changes for the merge. The default would be fine in this case, though it is just sample code.

As for the nesting comment, I have a work item for more design on formatting APIs. I'll address this item when I go that work item.